docs/search.md

```diff
@@ -94,6 +94,16 @@ The search button displays in the bottom right corner of the screen and triggers
 search.button: true
 ```
 
+### Focus search bar with ctrl + <focus_shortcut_key>
+
+The search input can be focused by pressing `ctrl + <provided_key>` (or `cmd + <provided_key>` on macOS).
+In case you want to disable this feature, set the following option to `false`.
+
+```yaml
+# Focus the search input by pressing `ctrl + <provided_key>` (or `cmd + <provided_key>` on macOS)
+focus_shortcut_key: 'k'
+```
+
```

Comment:
Many of the angle-bracket-delimited items here are being interpreted as potential HTML tags. That's flagging the HTML validator in CI + is probably not best practice.

I also have some minor nits re: writing style & using semantic HTML, and explaining what a valid input is.

I'm thinking something like this, what are your thoughts?

````suggestion
### Focus search bar with a keyboard shortcut

Just the Docs supports focusing the search bar input with a keyboard shortcut. After setting the `search.focus_shortcut_key` config item key, users who press <kbd>Ctrl</kbd> + `search.focus_shortcut_key` (or on macOS, <kbd>Command</kbd> + `search.focus_shortcut_key`) will focus the search bar.

Note that this feature is **disabled by default**. `search.focus_shortcut_key` should be a [valid value from `KeyboardEvent.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key); this involves all ASCII alphanumeric values, as well as modifier keys.

For example,

```yaml
search:
    focus_shortcut_key: 'k'
```

Will make <kbd>Ctrl</kbd> + <kbd>K</kbd> focus the search bar for Windows users (and <kbd>Command</kbd> + <kbd>K</kbd> on macOS).

````